packages/fleet/fleet-agent/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/fleet/releases/download/v0.3.11/fleet-agent-0.3.11.tgz
-version: 100.0.5
+url: https://github.com/rancher/fleet/releases/download/v0.4.0-rc1/fleet-agent-0.4.0-rc1.tgz
+version: 100.0.6
```

Comment:
I have one question:
Should the chart version be bumped to `100.1.0` because Fleet bumps its minor version? 
 
@aiyengar2 do you know the answer?

Reply:
That sounds good; then it depends on the Fleet team :)  
cc @mattfarina

Reply:
I moved to `100.1.0`. It's ready for a review.